src/main/org/deidentifier/arx/ARXLattice.java

```diff
@@ -1265,6 +1263,13 @@ private void readObject(ObjectInputStream aInputStream) throws ClassNotFoundExce
         }
     }
     
+    private void setMonotonicity(boolean isSuppressionAlwaysEnabled, int absoluteMaxOutliers) {
```

Comment:
I'm also not sure that the changes in ARXLattice are correct. The affected logic is pretty complex and I'm having a hard time remembering exactly how this works... :)

It is not used for optimizing the search process (the ARXLattice is always constructed after the search process) but for estimating the data quality/utility/info.-loss of transformations which have been pruned during the search process (the estimation is implemented in class UtilityEstimator, which also contains a (short) description of the process). I will try to wrap my head around it.